config/sites/classrooms.uiowa.edu/config_split.patch.user.role.webmaster.yml

```diff
@@ -2,18 +2,34 @@ adding:
   dependencies:
     config:
       - node.type.room
+      - taxonomy.vocabulary.room_features
+      - taxonomy.vocabulary.technology_features
+      - taxonomy.vocabulary.units
+    module:
+      - rabbit_hole
   permissions:
     - 'create room content'
+    - 'create terms in room_features'
+    - 'create terms in technology_features'
+    - 'create terms in units'
     - 'delete any room content'
     - 'delete own room content'
     - 'delete room revisions'
+    - 'delete terms in room_features'
+    - 'delete terms in technology_features'
+    - 'delete terms in units'
     - 'edit any room content'
     - 'edit own room content'
+    - 'edit terms in room_features'
+    - 'edit terms in technology_features'
+    - 'edit terms in units'
     - 'enter room revision log entry'
     - 'override room authored by option'
     - 'override room authored on option'
     - 'override room published option'
     - 'override room sticky option'
+    - 'rabbit hole administer taxonomy_term'
```

Comment:
```suggestion
```

we don't want webmasters to configure rabbit hole. just want them to bypass any blocked/redirected paths associated with RH